src/main/java/CovidLoveit/Domain/Models/Guideline.java

```diff
@@ -204,4 +204,12 @@ public Industry getIndustry() {
     public void setIndustry(Industry industry) {
         this.industry = industry;
     }
+
+    public Date getCreatedAt() {
```

Comment:
1. Do you want people to be able to edit the createdAt timestamp?
2. You can include an annotation @CreationTimestamp if you want the attribute to be auto-generated